internal/util/projutil/project_util.go

```diff
@@ -215,8 +215,15 @@ func GetOperatorType() OperatorType {
 }
 
 func IsOperatorGo() bool {
-	_, err := os.Stat(mainFile)
-	return err == nil
+	_, errGoMod := os.Stat(goModFile)
+	_, errGopkgTOMLFile := os.Stat(gopkgTOMLFile)
+	_, errPkgApiDir:= os.Stat(pkgApiDir)
+
+	// check fif has go modules or dep files or pkg/api
```

Comment:
typo: fif -> if
rephrase: check if we have go modules, dep files or pkg/api

Reply:
👍